src/components/popover/Popover.js

```diff
@@ -20,13 +20,11 @@ class Popover extends PureComponent {
 
   componentDidMount() {
     document.body.appendChild(this.popoverRoot);
-
-    events.addEventsToWindow({ resize: this.setPlacementThrottled, scroll: this.setPlacementThrottled });
+    events.addEventsToWindow({ scroll: this.setPlacementThrottled });
```

Comment:
Do yo still need to add the scroll event? Because it seems to be handled by the `ReactResizeDetector as well`?

Reply:
Yes we still do need this. I added the onScroll prop to the `ReactResizeDetector` as a quick test and just forgot to remove it again (since it has no effect).